src/main/java/seedu/address/logic/parser/ListTodayPrescriptionCommandParser.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.MessagesPrescription.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.ListTodayPrescriptionCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ListTodayPrescriptionCommand object
+ */
+public class ListTodayPrescriptionCommandParser implements ParserPrescription<ListTodayPrescriptionCommand> {
```

Comment:
This class ended up not being used. Can use it in the PrescriptionListParser to check for invalid commands 

So in PrescriptionListParser it will end up becoming smt like

new ListTodayPrescriptionCommandParser().parse(args)